interni/norme-di-progetto/components/processi-di-supporto/documentazione.tex

```diff
@@ -49,12 +52,80 @@ \subsubsection{Implementazione}%
   \item [Archiviazione] l'amministratore archivia il documento in un repository pubblico su \glossario{GitHub}.
 \end{description}
 
+\paragraph{Documenti ufficiali}%
+\label{par:documenti_ufficiali}
+
+In questa sezione verranno indicati tutti i documenti che GruppOne si impegna a consegnare completi entro la revisione di accettazione. Per ogni documento saranno indicati nome del documento, scopo del documento e destinatari.
+I documenti ufficiali e prodotti finali del processo di documentazione sono i seguenti:
+
+\begin{itemize}
+  \item Analisi dei requisiti
+
+        \textit{Scopo}: descrivere i requisiti e comprendere le funzionalità che il sistema software dovrà offrire.
+
+        \textit{Destinatari}: tutti i componenti del team e committente.
```

Comment:
Penso sia meglio scrivere "ed il committente". Se si fa la correzione, va sistemato anche nei punti seguenti in cui si usa la forma